pkg/kubeadm/master/addons.go

```diff
@@ -198,6 +198,16 @@ func createKubeDNSPodSpec(params *kubeadmapi.BootstrapParams) api.PodSpec {
 	}
 
 }
+func createKubeDNSServiceSpec(params *kubeadmapi.BootstrapParams) api.ServiceSpec {
+	return api.ServiceSpec{
+		Selector: map[string]string{"name": "kube-dns"},
+		Ports: []api.ServicePort{
+			{Name: "dns", Port: 53, Protocol: api.ProtocolUDP},
+			{Name: "dns-tcp", Port: 53, Protocol: api.ProtocolTCP},
+		},
+		ClusterIP: "100.64.0.2",
```

Comment:
Ok. I was thinking to make it `.53`, but I'm equally happy with `.10`, and trust you've thought about this. I picked `.2` just to cause I had to pick something.